presto-raptor/src/main/java/com/facebook/presto/raptor/storage/organization/ShardIndexInfo.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.raptor.storage.organization;
+
+import java.util.Objects;
+import java.util.Optional;
+import java.util.OptionalInt;
+import java.util.UUID;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+public class ShardIndexInfo
+{
+    private final long tableId;
+    private final OptionalInt bucketNumber;
+    private final UUID shardUuid;
+    private final long rowCount;
+    private final long uncompressedSize;
+    private final Optional<ShardRange> shardRange;
```

Comment:
I think this would be more clear as `sortRange`. It took me a while to figure out what was going on here